File: Metode-Avansate-de-Programare/lab2/model/StackContainer.java
package sem1_2.model;

import java.util.ArrayList;
import java.util.List;
import java.util.Stack;

public class StackContainer extends ArrayContainer {
    public StackContainer() {}
    @Override
    public Task remove() {
        if (!tasks.isEmpty()) {
            return tasks.remove(tasks.size() - 1);
        } else {
            return null;
        }
    }

}

File: Metode-Avansate-de-Programare/lab11/lab11/StackContainer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace lab11;
public class StackContainer :AbstractContainer
{
    public override Task remove()
    {
        if (!base.isEmpty())
        {
            Task t = tasks.ElementAt(tasks.Count() - 1);
            tasks.RemoveAt(tasks.Count() - 1);
            return t;
        }
        else
        {
            return null;
        }
    }

}
